astitodo.go

```diff
@@ -22,8 +23,11 @@ var (
 	todoIdentifiers = []string{"TODO", "FIXME"}
 )
 
-// TODOs represents a set of todos
-type TODOs []*TODO
+// TODOContainer represents a set of todos
```

Comment:
I don't really like the idea of creating this wrapper here.
I'd rather keep the old `TODOs` struct and if you need the root path in one of the `Write` func, you inject it as one of the method argument.